RUFAS/routines/manure/manure_manager.py

```diff
@@ -9,6 +9,7 @@
 from RUFAS.time import Time
 
 from ...data_structures.pen_manure_data import PenManureData
+from ...output_manager import OutputManager
```

Comment:
```suggestion
from RUFAS.output_manager import OutputManager
```

I believe we want to get away from these relative import statements when possible.

Reply:
Hmm we should ask @PooyaHekmati about this, I was under the impression they were a best practice but I'm not sure.

Reply:
@Isgotkowitz I'm curious to know the reason.

Reply:
@PooyaHekmati in the case that the path to module is modified (something gets renamed), the number of import statements that have to be modified is minimized.

Reply:
I don't think so. The relative path to the module is different depending on where you are importing it from. Therefore, when it needs modification, multiple things have to be modified if the paths are relative. But if all the paths to OM are absolute, then it is as easy as bulk replacement. 
Also, this is anti-pattern, which is bad.

Reply:
Noted! I will make this absolute then, thanks Pooya